app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -48,6 +49,9 @@ class SelectionInteractionViewModel(
       ?: minAllowableSelectionCount
   }
   private val selectedItems: MutableList<Int> = mutableListOf()
+
+  /**It stores the number of selected answers. It updates on answer selection updates.*/
```

Comment:
```suggestion
  /** The number of currently selected answers. */
```

Rewording to make it a bit more concise.